tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -113,6 +113,12 @@ func newSession(credentialPath, credentialAccountID, region string) (*awssession
 	})
 }
 
+// CreateWindowsVMWithPrivate creates a WindowsVM with private IP
+func (a *AwsProvider) CreateWindowsVMWithPrivateIP() (windowsVM types.WindowsVM, err error) {
```

Comment:
Shouldn't this be `CreateWindowsVMWithoutPublicIP()` a private IP will be created no matter what

Reply:
It should be `CreateWindowVMWithPrivateSubnet()` :-)

Reply:
Done